README.md

```diff
@@ -195,6 +199,66 @@ When an OdeBsmJson message is processed through the jpo-geojsonconverter, a Proc
 }
 ```
 
+### ProcessedPsm
+
+When an OdePsmJson message is processed through the jpo-geojsonconverter, a ProcessedPsm message is created. This message is a single JSON object that contains all of the important information within a PSM message for matching it to a corresponding ProcessedMap message and for identifying its state. There is no geoJSON component to a ProcessedMap message on its own.

```

Comment:
> This message is a single JSON object that contains all of the important information within a PSM message for matching it to a corresponding ProcessedMap message and for identifying its state. There is no geoJSON component to a ProcessedMap message on its own.

I don't think this makes sense for PSMs.  They aren't necessarily associated with intersections or MAPs, and are geoJSON.

Reply:
Yep, that is definitely incorrect. Let me update this.

Reply:
addressed